app/assets/javascripts/controllers/home/home.coffee

```diff
@@ -1 +1,8 @@
 @cooking.controller 'homeCtrl', ($scope) ->
+  $scope.recipes = [
+    {
+      id:1,
+      title:"recipe 1",
+      description:"SmartOS images come with pkgsrc pre-installed. On other illumos distributions, first install pkgsrc, then you may install the binary package as normal: SmartOS images come with pkgsrc pre-installed. On other illumos distributions, first install pkgsrc, then you may install the binary package as normal:",
+      username:"Mohamed Belaid"
+      }]
```

Comment:
Donia created a new branch to put the necessary factory that return our recipes objects.